_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -0,0 +1,191 @@
+    patent] and so to avoid any legal uncertainty, the implementation has not previously been
+    distributed to users. On September 25, the patent expired, removing that legal
+    uncertainty.
+    [A PR has been opened in the libsecp256k1 repo][endomorphism PR] to
+    always use the GLV endomorphism algorithm, which is expected to decrease
+    Bitcoin Core's initial sync time significantly.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What is the history of nLockTime in Bitcoin Core?]({{bse}}90229)
+  Reviewing older versions of Bitcoin Core's source code, Martin Harrigan asks
+  for clarification on whether height-based or time-based nLockTime features
+  were added first and what fork implications that had. Pieter Wuille and David A. Harding
+  confirm height-based logic came first and give some interesting historical insights.
+
+- [Is there any other P2P protocol in use besides a "Gossip protocol"?]({{bse}}99131)
+  Murch outlines the different communication protocols used in the Bitcoin
+  ecosystem including [Bitcoin's P2P protocol][bitcoin p2p messages], mining
+  protocols like FIBRE and stratum, LN-related protocols, as well as multiparty
+  coordination protocols like [payjoin][topic payjoin].
+
+- [Why do Anchor Outputs need to enforce an nSequence of 1?]({{bse}}98848)
+  Dalit Sairio asks about [anchor outputs][topic anchor outputs] and the
+  necessity of the `OP_CHECKSEQUENCEVERIFY` (CSV) addition to LN scripts. User
+  darosior describes the concerns around [CPFP carve-out][topic cpfp carve out]
+  and why the delay of 1 block is needed.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta.rc4][lnd 0.11.1-beta] is the release candidate for a
+  minor version.  Its release notes summarize the changes as "a number
+  of reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor commitments][topic
+  anchor outputs] spec compliant."
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
```

Comment:
I'm not sure why, but this seems to be causing the travis build to fail: https://travis-ci.org/github/bitcoinops/bitcoinops.github.io/builds/731360581#L713, but seems fine in the netlify build

Reply:
fixed by rebasing and resolving the silent merge conflict